ix-dev/community/palworld/templates/docker-compose.yaml

```
@@ -130,6 +139,7 @@ services:
         set_ini_value "ServerDescription" "{{ values.palworld.server.description }}" true
         set_ini_value "ServerPassword" '{{ values.palworld.server.password }}' true false
         set_ini_value "AdminPassword" '{{ values.palworld.admin_password }}' true false
+        set_ini_value "AllowConnectPlatform" '{{ values.palworld.allow_platform }}' false true
```

```suggestion
        set_ini_value "AllowConnectPlatform" '{{ values.palworld.allow_platform }}' false
```
print defaults to true anyway!